lessons/Data-Science/box-plots/langs/en-us/pages/match-dot-plots-summaries.adoc

```diff
@@ -0,0 +1,87 @@
+= Matching Dot Plots and Five-Number Summaries
+
+++++
+<style>
+/* Format matching answers to render with an arrow */
+.solution::before{ content: ' → '; }
+</style>
+++++
+Draw a line from each dot plot on the left to the corresponding five-number summary on the right.
```

Comment:
I was expecting the solution page version of this to be annotated as you propose in the lesson plan. Which also makes me wonder whether the directions should encourage students to annotate, and then match.